eng/pipelines/templates/jobs/ci.mgmt.yml

```diff
@@ -68,7 +68,7 @@ jobs:
           OSVmImage: windows-2022
         MacOs:
           Pool: Azure Pipelines
-          OSVmImage: "macos-11"
+          OSVmImage: "macos-latest"
```

Comment:
@ArthurMa1978 @live1206 is this pipeline needed any longer? It has been failing for at least a month.

Reply:
Yes, it is still needed for sdk-for-net PR CI. I think it worked last week [here](https://github.com/Azure/azure-sdk-for-net/pull/44576/checks?check_run_id=26245500591), got ignored quite recently.
Thanks for the update to make it work again!

Reply:
I don't see a passing build since the beginning of May https://dev.azure.com/azure-sdk/public/_build?definitionId=529&_a=summary

Reply:
Sorry, I just realized this is for Track 1 mgmt SDK, then this is not needed anymore.
And it started to fail because there are some moderate severity vulnerability dependent packages.